tool/lib/can_actions.py

```diff
@@ -120,11 +120,16 @@ def send(self, data, arb_id=None, is_extended=None, is_error=False, is_remote=Fa
         # Force extended flag if it is unspecified and arbitration ID is larger than the standard format allows
         if is_extended is None:
             is_extended = arb_id > ARBITRATION_ID_MAX
+        if pad:
+            data+=[0]*(8-len(data))
+        print(data)
```

Comment:
We _really_ should not force printing of every message sent through this function.